src/cms/models/events/event.py

```diff
@@ -12,42 +12,44 @@
 class Event(models.Model):
     """
     Data model representing an event.
-
-    :param id: The database id of the event
-    :param start_date: The date when the event starts
-    :param start_time: The time when the event starts
-    :param end_date: The date when the event ends
-    :param end_time: The time when the event ends
-    :param picture: The thumbnail image of the event
-    :param archived: Whether or not the event is archived
-
-    Relationship fields:
-
-    :param region: The region to which the event belongs (related name: ``events``)
-    :param location: The point of interest where the event takes place (related name: ``events``)
-    :param recurrence_rule: If the event is recurring, the recurrence rule contains all necessary information on the
-                            frequency, interval etc. which is needed to calculate the single instances of a recurring
-                            event (related name: ``events``)
-
-    Reverse relationships:
-
-    :param translations: The translations of this event
-    :param feedback: The feedback to this event
+    Can be direcly imported from :mod:`cms.models`.
     """
 
-    region = models.ForeignKey(Region, related_name="events", on_delete=models.CASCADE)
+    region = models.ForeignKey(
+        Region,
+        on_delete=models.CASCADE,
+        related_name="events",
+        verbose_name="The region to which the event belongs",
+    )
     location = models.ForeignKey(
-        POI, related_name="events", on_delete=models.PROTECT, null=True, blank=True
+        POI,
+        null=True,
+        blank=True,
+        on_delete=models.PROTECT,
+        related_name="events",
+        verbose_name="The point of interest where the event takes place",
+    )
+    start_date = models.DateField(verbose_name="The date when the event starts")
+    start_time = models.TimeField(
+        blank=True, verbose_name="The time when the event starts"
     )
-    start_date = models.DateField()
-    start_time = models.TimeField(blank=True)
-    end_date = models.DateField()
-    end_time = models.TimeField(blank=True)
+    end_date = models.DateField(verbose_name="The date when the event ends")
+    end_time = models.TimeField(blank=True, verbose_name="The time when the event ends")
     recurrence_rule = models.OneToOneField(
-        RecurrenceRule, null=True, on_delete=models.SET_NULL
+        RecurrenceRule,
+        null=True,
+        on_delete=models.SET_NULL,
+        help_text="If the event is recurring, the recurrence rule contains all necessary information on the frequency, interval etc. which is needed to calculate the single instances of a recurring event",
```

Comment:
The documentation says the following:
- `verbose_name`: A human-readable name for the field. If the verbose name isn’t given, Django will automatically create it using the field’s attribute name, converting underscores to spaces - [Source](https://docs.djangoproject.com/en/3.1/ref/models/fields/#verbose-name)
- `help_text`: Extra “help” text to be displayed with the form widget. It’s useful for documentation even if your field isn’t used on a form. - [Source](https://docs.djangoproject.com/en/3.1/ref/models/fields/#help-text)

So I tried to follow that principles, my rule of thumb is:
- `verbose_name` is a noun with an optional genitive construction, e.g. "The thumbnail image of the event"
- `help_text` is a full sentence explaining the field, its purpose or any other useful context info.

Even if we don't use the inbuilt Django functionality which makes use of these field attributes, they are really helpful for our documentation. The advantage of using these attributes instead of writing the text in docstrings is that I can build the documentation automatically based on the fields which leaves less room for errors like e.g. using the wrong types, forgetting to change fields in the docstring when they are changed in the models, using the wrong related names and so on. The docstrings of model fields are now generated by [sphinxcontrib-django2](https://pypi.org/project/sphinxcontrib-django2/).

Reply:
Seems plausible to me, thanks for elaborating.